tests/tests_app_examples/test_multi_node.py

```diff
@@ -27,3 +27,27 @@ def test_multi_node_example():
     result = application_testing(LightningTestMultiNodeApp, command_line)
```

Comment:
Can we add a test on the number of works that get created/started based on the value of the `num_nodes` argument?

Reply:
Done !